src/components/tooltip/Tooltip.tsx

```diff
@@ -3,7 +3,7 @@ import { StyledTooltip, TooltipWrapper } from "./styles";
 import { useTooltipHover } from "./hooks";
 
 export type Props = {
-  caption: string;
+  caption: string | React.ReactNode;
```

Comment:
ReactNode includes string

```suggestion
  caption: React.ReactNode;
```